dataeng/jobs/analytics/SnowflakeRefreshSnowpipe.groovy

```diff
@@ -0,0 +1,81 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+
+
+// If you want to configure another table to a Snowpipe, create a config map
+// here and add it to the jobConfigs list.
+Map eventSnowpipeConfig = [
+    NAME: 'refresh-snowpipe-event-loader',
+    PIPE_NAME: 'snowpipe_event_loader',
+    TABLE_NAME: 'json_event_records'
+]
+List jobConfigs = [
+    eventSnowpipeConfig
+]
```

Comment:
+1 on putting more job configgy stuff in the DSL rather than in analytics-secure.  It's tiresome to hop back and forth just to see what a job does.